src/ducks/connections/index.js

```diff
@@ -0,0 +1,42 @@
+import konnector from './konnector'
+
+// constant
+export const CREATE_CONNECTION = 'CREATE_CONNECTION'
+export const UPDATE_CONNECTION_RUNNING_STATUS = 'UPDATE_CONNECTION_RUNNING_STATUS'
+
+// reducers
+
+const reducer = (state = {}, action) => {
+  switch (action.type) {
+    case CREATE_CONNECTION:
+    case UPDATE_CONNECTION_RUNNING_STATUS:
+      const konnectors = { ...state }
+      konnectors[action.konnector.slug] = konnector(konnectors[action.konnector.slug], action)
+      return konnectors
+    default:
+      return state
+  }
+}
+
+export default reducer
+
+// action creators sync
+export const createConnection = (konnector, account, folder) => ({
+  type: CREATE_CONNECTION,
+  konnector: konnector,
+  account: account,
+  folder: folder
+})
```

Comment:
There is a short way to write this:

```js
export const createConnection = (konnector, account, folder) => ({
  type: CREATE_CONNECTION,
  konnector,
  account,
  folder
})
```

Reply:
Kenw this one, but it is not yet a reflex. Thanks 👍